releases/release-6.1.1.md

```diff
@@ -0,0 +1,176 @@
+    (dup: release-6.2.0.md > Bug fixes> TiKV)- Fix the panic issue that might occur when the source peer catches up logs by snapshot in the Region merge process [#12663](https://github.com/tikv/tikv/issues/12663)
+    (dup: release-5.3.2.md > Bug Fixes> TiKV)- Fix the issue of frequent PD client reconnection that occurs when the PD client meets an error [#12345](https://github.com/tikv/tikv/issues/12345)
+    - Fix encryption keys not cleaned up when Raft Engine is enabled [#13123](https://github.com/tikv/tikv/issues/13123)
+    (dup: release-6.2.0.md > Bug fixes> TiKV)- Fix the issue that the Commit Log Duration of a new Region is too high, which causes QPS to drop [#13077](https://github.com/tikv/tikv/issues/13077)
+    - Fix a rare case panic when Raft Engine is enabled [#12698](https://github.com/tikv/tikv/issues/12698)
+    - Avoid redundant log warnings when procfs is not available [#13116](https://github.com/tikv/tikv/issues/13116)
+    - Fix the wrong expression of `Unified Read Pool CPU` in dashboard [#13086](https://github.com/tikv/tikv/issues/13086)
+    - Make default `region-split-check-diff` not less than bucket size [#12598](https://github.com/tikv/tikv/issues/12598)
+    - Fix panics when apply snapshot is aborted and Raft Engine is enabled [#12470](https://github.com/tikv/tikv/issues/12470)
+    - Refactor PD client to avoid potential deadlock(RWR) [#13191](https://github.com/tikv/tikv/issues/13191)
+
++ PD
+
+    - Fix the issue that the online process is not accurate when having invalid label settings [#5234](https://github.com/tikv/pd/issues/5234)
+    - Fix the problem that gRPC handles return errors inappropriately [#5373](https://github.com/tikv/pd/issues/5373)
+    - Fix the issue that `/regions/replicated` may return the wrong status [#5095](https://github.com/tikv/pd/issues/5095)
+
++ TiFlash
+
+    (dup: release-5.4.2.md > Bug Fixes> TiFlash)- Fix the issue that TiFlash crashes after dropping a column of a table with clustered indexes in some situations [#5154](https://github.com/pingcap/tiflash/issues/5154)
+    - Fix the issue that the `format` function might return a data truncated error [#4891](https://github.com/pingcap/tiflash/issues/4891)
+    - Fix the issue that some obsolete data might persist in storage and cannot be deleted [#5659](https://github.com/pingcap/tiflash/issues/5659)
+    - Fix unnecessary CPU usage in some edge cases [#5409](https://github.com/pingcap/tiflash/issues/5409)
+    - Fix a bug that TiFlash cannot work in a cluster using IPv6 [#5247](https://github.com/pingcap/tiflash/issues/5247)
+    - Fix a bug that TiFlash might crash due to an error in parallel aggregation [#5356](https://github.com/pingcap/tiflash/issues/5356)
+    - Fix a bug that thread resources might leak in case of MinTSOScheduler query errors [#5556](https://github.com/pingcap/tiflash/issues/5556)
```

Comment:
```suggestion
    - Fix a bug that thread resources might leak in case of `MinTSOScheduler` query errors [#5556](https://github.com/pingcap/tiflash/issues/5556)
```